.github/workflows/relase_modules.yml

```diff
@@ -0,0 +1,46 @@
+name: Changelog
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .changeset/** # Only on changes to ./changeset/*
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  changelog:
+    name: Update CHANGELOG
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout code
+          uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+          with:
+            # https://github.com/actions/checkout/issues/1471#issuecomment-1771231294
+            fetch-tags: 'true'
+            fetch-depth: 0
+
+        # Corepack is an official tool by Node.js that manages package managers versions
+        - name: Setup yarn
+          run: corepack enable
+
+        - name: Setup Node.js environment
+          uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+          with:
+            node-version: 18
```

Comment:
**question**: why not using the version of the [`.node-version` file](https://github.com/pagopa/dx/blob/main/.node-version)?

The [`setup-node` action allows you to use the `node-version-file` option](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file), which gets the node version from the specified file.